libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts

```diff
@@ -253,6 +258,36 @@ describe('DropdownComponent', () => {
       });
     });
 
+    describe('when configured with open direction', () => {
+      it('should render card to the right', () => {
+        spectator.component.open = HorizontalDirection.right;
+        spectator.component['state'] = OpenState.open;
+        spectator.detectChanges();
+
+        const buttonRect = buttonElement.getBoundingClientRect();
+        const card = spectator.query('kirby-card');
+        const cardRect = card.getBoundingClientRect();
+
+        expect(cardRect.left).toEqual(buttonRect.left);
+      });
+
+      it('should open card to the left', (done) => {
+        spectator.component.open = HorizontalDirection.left;
+        spectator.element.style.cssFloat = 'right';
+        spectator.component['state'] = OpenState.open;
+        spectator.detectChanges();
+
+        setTimeout(() => {
+          const card = spectator.query('kirby-card');
+          const buttonRect = buttonElement.getBoundingClientRect();
+          const cardRect = card.getBoundingClientRect();
+
+          expect(cardRect.right).toEqual(buttonRect.right);
+          done();
+        }, openDelayInMs);
+      });
+    });
```

Comment:
Consider align tests if possible - any reason why we need to do this 2 different ways? 🤔

Reply:
There was at the time :) -  I think it was related to how I initially set the direction in the intersectionObserver, so I needed the timeout. But now it works fine without